pagopa/pn-frontend
# fix: upgrade sonarqube-scanner and remove danger

## Short description
Upgrades sonarqube-scanner and removes danger-plugin-digitalcitizenship from dependencies in personagiuridica-webapp

## List of changes proposed in this pull request
- Upgrades sonarqube-scanner in personagiuridica-webapp
- Removes danger-plugin-digitalcitizenship from dependencies in personagiuridica-webapp

## How to test
It shouldn't affect apps behavior and pipelines, so yarn start build and test should work correctly, pipelines should run correctly too.
